src/data/set/lattice.lean

```diff
@@ -1649,3 +1649,13 @@ noncomputable def Union_eq_sigma_of_disjoint {t : α → set β}
 (equiv.of_bijective _ $ sigma_to_Union_bijective t h).symm
 
 end set
+
+open set
+
+variables [complete_lattice β]
+
+lemma supr_Union (s : ι → set α) (f : α → β) : (⨆ a ∈ (⋃ i, s i), f a) = ⨆ i (a ∈ s i), f a :=
```

Comment:
This feel like a more accurate name
```suggestion
lemma bsupr_Union (s : ι → set α) (f : α → β) : (⨆ a ∈ (⋃ i, s i), f a) = ⨆ i (a ∈ s i), f a :=
```

Reply:
This does not match `supr_union`

Reply:
There are many more lemmas like that, for example `supr_insert`, `supr_finset_image`... so it's definitely not for this PR.

Reply:
I'm not really convinced either, because the point of adding the `b` is to disambiguate but there is no ambiguity here in the first place. It would just make the names one letter longer.